app/server/authSetup.js

```diff
@@ -1,9 +1,12 @@
 import passport from 'passport';
 import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
-import session from 'express-session';
+import cookieSession from 'cookie-session';
 
 export const authSetup = app => {
-  app.use(session({ secret: process.env.SESSION_SECRET, resave: true, saveUninitialized: true }));
+  app.use(cookieSession({
+    secret: process.env.SESSION_SECRET,
```

Comment:
I would say 2 weeks, but reading again I see these are session cookies, not persistent cookies. So don't mind me, it's fine as is!